R/highlight_function.R

```diff
@@ -0,0 +1,55 @@
+#' Highlight palette prep function
+#'
+#' Create named list to serve as palette input for following fill/color highlight functions
+#'
+make_highlight_palette <- function(variable, highlight, color = "#009ccc"){
```

Comment:
I think arg defaults in the subfunction are unnecessary and potentially confusing, as the defaults are already set in the arguments of the top-level functions.